resources/views/pages/profile.blade.php

```diff
@@ -0,0 +1,41 @@
+@extends('home')
+@section('css')
+    <link rel="stylesheet" href="{{ asset('bower_components/review_travel/css_travel/profile.css') }}">
+    <link rel="stylesheet" href="{{ asset('bower_components/review_travel/css_travel/edit_profile.css') }}">   
+@endsection
+@section('content')
+<div class="ui-bg-cover ui-bg-overlay-container text-white bradcam_bg_4" id="image_of_title">
+    <div class="ui-bg-overlay bg-dark opacity-50 "></div>
+    <div class="container py-5">
+        <div class="media mx-auto">
+            @if (Auth::user()->avatar == NULL)
+                <img src="{{ asset(config('constains.avatar')) }}" 
+                    class="avt_image rounded-circle" alt="User avatar" id="avt_img"></label>
```

Comment:
```suggestion
                    class="avt-image rounded-circle" alt="User avatar" id="avt-img"></label>
```
class và id dùng gạch này nhé.
i18n ở alt